app/models/event/question.rb

```diff
@@ -20,21 +22,21 @@
 class Event::Question < ActiveRecord::Base
 
   belongs_to :event
-
   has_many :answers, dependent: :destroy
 
   validates_by_schema
-  validate :assert_zero_or_more_than_one_choice
+  validate :assert_number_of_choices
```

Comment:
Als User ist die aktuelle Lösung sehr verwirrend und eher ein Hack, dass wir es auf die `multiple_choices` abbilden. Ich schlage folgende Lösung vor, die einfacher und auch verständlicher sein sollte:

- Wir machen keine DB Migration (kein `checkbox` Flag)
- Wir erlauben auch eine einzelne Antwort (`assert_zero_or_more_than_one_choice` erübrigt sich)
- Hilfetext bei den Antworten: hier muss unbedingt erklärt sein, was das Trennzeichen ist und dass eine oder mehrere Antworten erfasst werden können.
- Businesslogik:
  - Wenn keine Anwort: Textfeld
  - Wenn eine Antwort (die Option «Mehrfachauswahl» hat in diesem Fall keinen Effekt): einzelne Checkbox
  - Wenn mehrere Antworten und keine Mehrfachauswahl: Radios
  - Wenn mehrere Antworten und Mehrfachauswahl: Checkboxen

Beispiele:
  - Eine Antwort nicht obligatorisch: Vegetarier
  - Eine Antwort obligatorisch: Ich akzeptiere die AGBs